### ISSUE TITLE
How to generate 3D object with Materials

### ISSUE THREAD

**Issue by ishandutta04** [NVlabs/nvdiffrec#65]
I am following the mentioned steps but the code stops responding during validation. How to generate the `.obj` file along with the `.mtl` file? Does it automatically get created in the `out` folder upon the validation completes and is there a way to skip the validation and directly save the model?

**Comment by jmunkberg** [comment]
Yes, the .obj file is automatically generated, see https://github.com/NVlabs/nvdiffrec/blob/main/train.py#L646

You can skip validation by specifying 
`"validate" : false,`

in the config
